crates/sui-json-rpc-types/src/sui_move.rs

```diff
@@ -120,6 +128,7 @@ pub struct SuiMoveNormalizedModule {
     pub name: String,
     pub friends: Vec<SuiMoveModuleId>,
     pub structs: BTreeMap<String, SuiMoveNormalizedStruct>,
+    pub enums: BTreeMap<String, SuiMoveNormalizedEnum>,
```

Comment:
we unfortunately can't do this here due to backwards compatibility, but would it have been preferred to have these unioned into a single "data_type" or "types" field?

Reply:
Yea, would have been nice if we could, but would break backwards compatibility, so this is the next best thing :/

Reply:
good to know for other apis. We should fix this in graphql and anywhere else we expose this going forward

Reply:
Already fixed in GraphQL:

https://github.com/MystenLabs/sui/blob/ae6275f6957ac936caffe04e82b7762ba192c4f8/crates/sui-graphql-rpc/schema.graphql#L1956-L1979

Reply:
@amnn so are the structs and enum fields going to be deprecated and removed eventually from the schema?

Reply:
Ashok would definitely know more here, but my understanding is we would keep `structs` and `enums` as explicit fields, but there's the `datatypes` field that allows you to iterate over all user-defined types in the module (but you'd then need to handle the casting).

Reply:
That's right -- we chose to keep them so that for the cases where you knew what kind of type you were dealing with, you could take advantage of avoiding the type cast.